pkg/gofr/cmd.go

```diff
@@ -62,9 +76,17 @@ func (cmd *cmd) handler(path string) Handler {
 	return nil
 }
 
-func (cmd *cmd) addRoute(pattern string, handler Handler) {
+func (cmd *cmd) addRoute(pattern string, handler Handler, description string) {
```

Comment:
I would suggest a refactoring where you would simply pass a route struct to this function

Reply:
Okay @ccoVeille . I will try it!